DTS-STN/Alpha-Site
# Report a problem update

# Description
https://trello.com/c/W6wQcxMm/441-report-a-problem-list-items-would-not-be-announced-by-a-screen-reader-in-form-mode

- Move the bullet list and the "privacy statement" link to the top of the form so the screen reader will read the description first
- Add a title, and modify the contents in the bullet according to the new design
- The bullet list and the link should now live outside of the form

## Test Instructions

- Visit https://alphasite.dts-stn.com, scroll down to the bottom of the page, click on "Report a problem or a mistake on this page" button 
- Check if the change matches the design for both English and French

## Checklist
